doc/3/controllers/document/m-create/snippets/m-create.java

```diff
@@ -0,0 +1,23 @@
+
+
+
+    ConcurrentHashMap<String, Object> document1 = new ConcurrentHashMap<>();
+    ConcurrentHashMap<String, Object> document2 = new ConcurrentHashMap<>();
+    ConcurrentHashMap<String, Object> body = new ConcurrentHashMap<>();
+
+    body.put("key1", "value1");
+
+    document1.put("_id", "some-id1");
+    document1.put("body", body);
+
+    document2.put("_id", "some-id2");
+    document2.put("body", body);
+
+    final ArrayList<ConcurrentHashMap<String, Object>> documents = new ArrayList<>();
+    documents.add(document1);
+    documents.add(document2);
+
+    ConcurrentHashMap<String, ArrayList<Object>> response = kuzzle.getDocumentController().mCreate("nyc-open-data", "yellow-taxi", documents)
+    .get();
+
+    System.out.println("Successfully created " + response.get("successes").size() + " documents");
```

Comment:
Can you do the snippet test on something from the result as well as for the other methods?

Also can you add the result of the `println` in comment? 
It helps developer to better apprehend the structure of the response object they have. We try to do this for every method (Look the [Javascript SDK](https://github.com/kuzzleio/sdk-javascript/blob/master/doc/7/controllers/document/create/snippets/create.js) and [Doc](https://docs.kuzzle.io/sdk/js/7/controllers/document/create/#usage))